PubHub/PubHub.Common/ApiService/ApiConstants.cs

```diff
@@ -8,7 +8,7 @@ namespace PubHub.Common.ApiService
 {
     public static class ApiConstants
     {
-        public const string API_ENDPOINT = "ApiEnpoint";
+        public const string API_ENDPOINT = "ApiEndpoint";
```

Comment:
What is this for?

Reply:
It is the name of the value containing the API base url in the `appsettings.json` file 😊